app/src/main/java/com/github/se/polyfit/ui/screen/LoginScreen.kt

```diff
@@ -126,3 +151,10 @@ class Authentication(private val navigation: Navigation, private val context: Co
     }
```

Comment:
I just noticed that response might be null here:
```kotlin
val response = result.idpResponse
Log.i("LoginScreen", "response: $response")
if (result.resultCode == RESULT_OK) {
``` 
that it might be a good idea to if response is null to still log it as an error (line 149)

Reply:
It also might be a good idea to be consistent in the type of login used. Here you used information (login.i) and above debug. For similar cases